dockclean/containers.py

```diff
@@ -19,20 +19,23 @@ def stop_all_running_containers(logger):
 
 
 def remove_all_stopped_containers(logger):
-    print("Removing all stopped containers.")
+    logger.info("Removing all stopped containers.")
+
     client = docker.from_env()
-    containers = client.containers.list(filters={"status": "exited"})
-    count = 0
-    for container in containers:
-        try:
-            container.remove()
-            logger.info(f"Removed container -> {container.name}")
-            count += 1
-        except:
-            logger.error(
-                f"Exception occurred while removing container -> {container.name}"
-            )
-    print(f"Total {count} stopped containers removed.")
+    deleted_containers = client.containers.prune()
+
+    containers = deleted_containers['ContainersDeleted']
+
+    if containers is not None:
```

Comment:
I think we should keep the `count` var initially set to 0. But instead of incrementing it like before, we just update it to `count = len(containers)` if `containers` is not `None`